### ISSUE TITLE
Roll Table result text not being sent to Discord

### ISSUE THREAD

**Issue by dcode** [therealguy90/foundrytodiscord#32]
**Describe the bug**
The result from a table roll went to chat, but the message content did not.

**To Reproduce**
1. Go to a rollable table
2. Click on 'Roll'
3. Click on message in chat and click "Send to Main" if it wasn't a public roll.

**Expected behavior**
I expected the roll result text to go to chat.

**Screenshots**

This is what was sent.
<img width="604" alt="image" src="https://github.com/therealguy90/foundrytodiscord/assets/171574/e3cb6869-5363-41c2-b95d-7df46840288e">

This is how it appears in Foundry chat:
![image](https://github.com/therealguy90/foundrytodiscord/assets/171574/1bef351d-fb8c-4e38-8aca-9f8b0ece3830)

**Additional context**

HTML element:
```
<li class="chat-message message flexcol" data-message-id="I9Bb5CXMqvYYBncn" style="position: relative;">
    <header class="message-header flexrow">
        <h4 class="message-sender"><div class="avatar"><img src="icons/svg/mystery-man.svg" alt="Gamemaster"></div><span class="name-stacked"><span class="title">Gamemaster</span><span class="subtitle"></span></span></h4>
        <span class="message-metadata">
            <time class="message-timestamp">17m ago</time>
            
        <a aria-label="Additional Controls" class="chat-control" data-context-menu=""><i class="fas fa-ellipsis-vertical fa-fw"></i></a></span>


        <span class="flavor-text">Draws a result from the Caught Stealing from Arcane Acquisitions table</span>
    </header>
    <div class="message-content">
        <div class="table-draw" data-table-id="NEGhq4yMG5BcxGNA">
    <div class="dice-roll">
    <div class="dice-result">
        <div class="dice-formula">1d6</div>
        <div class="dice-tooltip" style="height: 0px;">
    <section class="tooltip-part">
        <div class="dice">
            <ol class="dice-rolls">
                <li class="roll die d6">5</li>
            </ol>
            <div class="total">
                <span class="label"></span>
                <span class="value">5</span>
            </div>
        </div>
    </section>
</div>

        <h4 class="dice-total">5</h4>
    </div>
</div>

    <ol class="table-results">
        <li class="table-result flexrow" data-result-id="fD2BGaMXESRAEU2g">
            <img class="result-image" src="icons/svg/d20-black.svg">
            <div class="result-text"><strong>Miser's Mark</strong>: Upon attempting to steal the item, a shimmering magical glyph appears on the character's forehead. It displays the stolen item's price tag in a glowing red font, visible to anyone who looks at them. The character becomes the walking advertisement for Flinar's wares, unable to remove the mark until the debt is paid.</div>
        </li>
    </ol>
</div>
    </div>
</li>
```

**Comment by dcode** [comment]
No errors in the console either.

<img width="655" alt="image" src="https://github.com/therealguy90/foundrytodiscord/assets/171574/f68ae99a-87ba-451b-93e1-14c0ec527c95">

**Comment by therealguy90** [comment]
This is something I forgot to implement. It shouldn't be too hard, but I'm going to be busy for a few days, so I won't be able to implement this right away. It won't take long once I find the time, though.

**Comment by therealguy90** [comment]
Fixed in 2.0.0.
